src/System.Private.CoreLib/shared/System/Activator.cs

```diff
@@ -15,48 +16,45 @@ public static partial class Activator
     {
         private const BindingFlags ConstructorDefault = BindingFlags.Instance | BindingFlags.Public | BindingFlags.CreateInstance;
 
+#if PROJECTN
+        [DebuggerHidden]
+        [DebuggerStepThrough]
+#endif
         public static object CreateInstance(Type type, BindingFlags bindingAttr, Binder binder, object[] args, CultureInfo culture) =>
             CreateInstance(type, bindingAttr, binder, args, culture, null);
 
+#if PROJECTN
```

Comment:
It should be fine for these attributes to be everywhere, without ifdef. Other similar reflection-invocation methods in shared partition, like `MethodBase.Invoke` have these attributes without any ifdefs.